lib/Steps/PromptForParameters.ts

```diff
@@ -120,7 +120,16 @@ export class PromptForParameters extends BaseStep {
   }
 
   private _validateAuthToken(input: string): boolean | string {
-    if (!input.match(/[0-9a-f]{64}/g)) {
+    const isOrgToken = input.startsWith('sntrys_');
+
+    if (isOrgToken) {
+      if (!isValidOrgToken(input)) {
+        return 'Make sure you copied the correct auth token, it should start with sntrys_ and consist of >140 base64 characters';
```

Comment:
Admittedly, this is a pretty bad hint message. @mydea can we say anything else about the structure of the token that would help users verify here?

Reply:
Ahh probably that the string needs to contain two `_` characters, no?

Reply:
yes, it must have two `_`, that should work as a check!

Reply:
I think mentioning that it has to start with `sntrys_` makes sense, as it is immediately obvious then when they copied something different.